data.tf

```diff
@@ -1,7 +1,9 @@
 locals {
   tfc_workload_identity_workspaces = distinct(flatten([
-    for org, workspaces in var.tfc_workload_identity_workspaces : [
-      for workspace in workspaces : "organization:${org}:project:${var.tfc_project_support_match}:workspace:${workspace}:run_phase:*"
+    for org, projects in var.tfc_workload_identity_workspaces : [
+      for project, workspaces in projects : [
+        for workspace in workspaces : "organization:${org}:project:${project}:workspace:${workspace}:run_phase:*"
```

Comment:
By default all existing workspaces would go to some default project, is it ?  since we are removing 

```
"organization:${org}:workspace:${workspace}:run_phase:*",
```

Reply:
I would require everyone to specify a project in the future, even if it is `Default Project`.